es/es-server/src/test/java/org/elasticsearch/common/settings/SettingsTests.java

```diff
@@ -615,4 +616,24 @@ public void testCopy() {
                                                     () -> Settings.builder().copy("not_there", settings));
         assertEquals("source key not found in the source settings", iae.getMessage());
     }
+
+    @Test
+    public void testGetAsStructuredMapNoSettingToMask() {
+        Settings settings = Settings.builder()
+            .put("masked", "masked_value")
+            .build();
+
+        Map<String, Object> map = settings.getAsStructuredMap(Set.of());
+        assertThat(map, hasEntry("masked", "masked_value"));
```

Comment:
This test will just test `getAsStructuredMap` as covered already by other tests, I don't see any need for it.

Reply:
There were no other tests in this file covering this, so I added it.
Also, it was a validation that the default behavior of `getAsStructuredMap` is not changed by my changes. 

For me, this is a valid test, no strong opinion though, I can remove it if you prefer.